cypress/objects/client.js

```diff
@@ -0,0 +1,56 @@
+class Client {
```

Comment:
This file could be called `ClientPage.js` because just "client" doesn't say much